build.gradle

```diff
@@ -10,34 +10,32 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
-        classpath 'com.gradle.publish:plugin-publish-plugin:0.10.1'
         classpath 'com.netflix.nebula:nebula-publishing-plugin:12.0.0'
-        classpath 'com.palantir.baseline:gradle-baseline-java:0.62.1'
-        classpath 'com.palantir.gradle.consistentversions:gradle-consistent-versions:1.9.2'
-        classpath 'gradle.plugin.com.palantir.gradle.gitversion:gradle-git-version:0.7.3'
-        classpath 'gradle.plugin.org.inferred:gradle-processors:2.2.0'
     }
 }
 
+plugins {
+    id 'com.jfrog.bintray' version '1.8.4' apply false
+    id 'com.gradle.plugin-publish' version '0.10.1'
+    id 'com.palantir.consistent-versions' version '1.9.2'
+    id 'org.inferred.processors' version '2.2.0'
+    id 'com.palantir.baseline' version '0.62.1'
+    id 'com.palantir.git-version' version '0.7.3'
+    id 'com.adarshr.test-logger' version '1.7.0'
```

Comment:
why com.adarshr.test-logger ?

Reply:
why not, provides pretty output for each test including how long it took, which I actually like